src/data/map.js

```diff
@@ -1140,7 +1141,7 @@ const getPortals = async (minLat, maxLat, minLon, maxLon, portalFilterExclude =
     WHERE lat >= ? AND lat <= ? AND lon >= ? AND lon <= ? ${sqlExcludeCreate}
     `;
     const args = [minLatReal, maxLatReal, minLonReal, maxLonReal];
-    const results = await dbManual.query(sql, args);
+    const results = await (dbOptions.includes('portal') ? db : dbManual).query(sql, args);
```

Comment:
We want to use manualdb for portals no? Especially since the `dbOptions` that's import is only for scanner tables?
```suggestion
    const results = await (dbOptions.includes('portal') ? dbManual : db).query(sql, args);
```

Reply:
Correct. Since the ternary will return false (assuming they set nests and portals to manualDb in config), then this will result in it reading from the manualDb.